src/python/foglamp/translators/omf_translator.py

```diff
@@ -0,0 +1,732 @@
+    """Creates the JSON data for OMF
+
+    Args:
+        target_stream_id:     OMF container ID
+        information_to_send:  information retrieved that should be prepared for OMF
+
+    Returns:
+        status: True if successful, False otherwise.
+        data_values_JSON: information converted in JSON format
+
+    Raises:
+        Exception: unable ro create the JSON message.
+
+    """
+
+    status = True
+    data_available = False
+
+    data_values_JSON = ''
+
+    row_id      = information_to_send.id
+    asset_code  = information_to_send.asset_code
+    timestamp   = information_to_send.user_ts.isoformat()
+    sensor_data = information_to_send.reading
+
+    debug_msg_write("INFO", "Stream ID : |{0}| ".format(target_stream_id))
+    debug_msg_write("INFO", "Sensor ID : |{0}| ".format(asset_code))
+    debug_msg_write("INFO", "Row    ID : |{0}| ".format(str(row_id)))
+
+    try:
+        # Prepares the data for OMF
+        data_values_JSON = [
+            {
+                "containerid": target_stream_id,
+                "values": [
+                    {
+                        "Time": timestamp
+                    }
+                ]
+            }
+        ]
+
+        #
+        # Evaluates which data is available
+        #
+        try:
+            data_values_JSON[0]["values"][0]["x"] = sensor_data["x"]
+            data_available = True
+        except:
+            pass
+
+        try:
+            data_values_JSON[0]["values"][0]["y"] = sensor_data["y"]
+            data_available = True
+        except:
+            pass
+
+        try:
+            data_values_JSON[0]["values"][0]["z"] = sensor_data["z"]
+            data_available = True
+        except:
+            pass
+
+        try:
+            data_values_JSON[0]["values"][0]["pressure"] = sensor_data["pressure"]
+            data_available = True
+        except:
+            pass
+
+        try:
+            data_values_JSON[0]["values"][0]["lux"] = sensor_data["lux"]
+            data_available = True
+        except:
+            pass
+
+        try:
+            data_values_JSON[0]["values"][0]["humidity"] = sensor_data["humidity"]
+            data_available = True
+        except:
+            pass
+
+        try:
+            data_values_JSON[0]["values"][0]["temperature"] = sensor_data["temperature"]
+            data_available = True
+        except:
+            pass
+
+        try:
+            data_values_JSON[0]["values"][0]["object"] = sensor_data["object"]
+            data_available = True
+        except:
+            pass
+
+        try:
+            data_values_JSON[0]["values"][0]["ambient"] = sensor_data["ambient"]
+            data_available = True
+        except:
+            pass
+
+
+
+        if data_available == True:
+            debug_msg_write("INFO", "OMF Message   |{0}| ".format(data_values_JSON))
+
+        else:
+            status = False
+            _log.warning(_message_list["e000009"])
+
```

Comment:
it is not clear, do you mean in this way : 


        if data_available:
            debug_msg_write("INFO", "OMF Message   |{0}| ".format(data_values_JSON))
        else:
            status = False
            _log.warning(_message_list["e000009"])

    except Exception as e: